app/models/miq_region.rb

```diff
@@ -62,7 +62,7 @@ def miq_servers
   end
 
   def servers_for_settings_reload
-    miq_servers.where(:status => "started")
+    MiqServer.in_my_region.where(:status => "started")
```

Comment:
Seems a little weird to be doing `.in_my_region` when everything else here does `.in_region(region_number)`, what about changing the configuration mixin to do `servers_for_settings_reload.in_my_region`?